### PR TITLE
Ocrvs - 4371

### PR DESCRIPTION
**Pull request by tahmidrahman-dsi**

### CODE REVIEW

File: packages/client/src/navigation/index.ts [hunk 1]
```diff
@@ -229,8 +229,10 @@ export function goToTeamSearch(searchedLocation?: searchedLocation) {
 }
 
 export function goToPerformanceHome(
-  timeStart: Date = startOfMonth(subMonths(new Date(Date.now()), 11)),
-  timeEnd: Date = new Date(Date.now()),
+  timeStart: Date = new Date(
+    startOfMonth(subMonths(new Date(Date.now()), 11)).setHours(0, 0, 0, 0)
+  ),
+  timeEnd: Date = new Date(new Date(Date.now()).setHours(0, 0, 0, 0)),
```
Comment (Zangetsu101) [inline packages/client/src/navigation/index.ts:235]: Would it not be better to use setHours(23, 59, 59)? That way the result will include the updated values for the current day

File: packages/client/src/utils/persistence/persistenceMiddleware.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+          'DECLARED',
+          'REJECTED',
+          'VALIDATED',
+          'WAITING_VALIDATION',
+          'REGISTERED'
+        ],
+        populationYear: new Date().getFullYear(),
+        officeSelected
+      },
+      fetchPolicy: 'cache-first'
```
Comment (Zangetsu101) [inline packages/client/src/utils/persistence/persistenceMiddleware.ts:48]: How about in the middleware we use `cache-and-network`, and `cache-first` in the views. That way when the app loads, we update the cache with new data if any and the views only query the cache

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]
@tahmidrahman-dsi Great work! Just a couple of things to think about

File: packages/client/src/utils/persistence/persistenceMiddleware.ts (hunk 2 above)
Comment (tahmidrahman-dsi) [reply packages/client/src/utils/persistence/persistenceMiddleware.ts:48]: Agreed, will update accordingly, thanks!

File: packages/client/src/navigation/index.ts (hunk 1 above)
Comment (tahmidrahman-dsi) [reply packages/client/src/navigation/index.ts:235]: Good spot @Zangetsu101, so only for `timeEnd` it should be like below, right?
```suggestion
  timeEnd: Date = new Date(new Date(Date.now()).setHours(23,59,59)),
```

Comment (Zangetsu101) [reply packages/client/src/navigation/index.ts]: yea this should work right?

Comment (tahmidrahman-dsi) [reply packages/client/src/navigation/index.ts]: Yes, it is done now

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**